Sources/StreamChat/APIClient/CDNClient/CDNClient.swift

```diff
@@ -4,15 +4,15 @@
 
 import Foundation
 
-/// API client that handles working with content (e.g. uploading attachments)
+/// The CDN client is responsible to upload files to a CDN.
 public protocol CDNClient {
-    /// Specifies maximum attachment size in bytes.
     static var maxAttachmentSize: Int64 { get }
 
+    /// Uploads attachment as a multipart/form-data and returns only the uploaded remote file.
     /// - Parameters:
-    ///   - attachment: An attachment to upload
-    ///   - progress: A closure that broadcasts upload progress
-    ///   - completion: Returns uploading result on upload completion or failure
+    ///   - attachment: An attachment to upload.
+    ///   - progress: A closure that broadcasts upload progress.
+    ///   - completion: Returns the file uploaded information.
```

Comment:
maybe "the uploaded file's information."?

Reply:
The only thing missing is the thumbnail for videos, which at the moment on iOS it is not a priority since we already have it locally. So a separate PR should handle that in the future